src/ui_dialogs.ts

```diff
@@ -108,80 +110,227 @@ export function openEditLinkDialog(column:model.LinkColumn, $header:d3.Selection
   });
 }
 
-export function sortDialog(column:model.HeatmapcustomColumn, $header:d3.Selection<model.HeatmapcustomColumn>) {
+export function sortDialogHeatmap(column: model.HeatmapcustomColumn, $header: d3.Selection<model.HeatmapcustomColumn>) {
 
-  console.log(column)
   var popup = makePopup($header, 'Sort By', `
     <form>
- 
-  
   <input type="radio" name="rank" id="min">Minimum<br>
-   <input type="radio" name="rank" id="max">Maximum<br>
-   <input type="radio" name="rank" id="mean">Mean<br>
+  <input type="radio" name="rank" id="max">Maximum<br>
+  <input type="radio" name="rank" id="mean">Mean<br>
   <input type="radio" name="rank" id="median">Median<br>
   <input type="radio" name="rank" id="q1">Q1<br>
   <input type="radio" name="rank" id="q3">Q3
-  
-  
-</form>
-
+    </form>
    `);
 
 
   popup.select('.ok').on('click', function () {
     var rank;
-if( popup.select("#min").property('checked')) {
-  rank = 'min';
-}
+    if (popup.select('#min').property('checked')) {
```

Comment:
you could assign to each input a corresponding value and use a css selector `:checked` to find the currently selected one. Or you could use d3 to render the list of radio buttons with a change handler. Both options avoid the long if/else cases